backend/src/webhook/webhook.controller.ts

```diff
@@ -0,0 +1,69 @@
+import {
+    Controller,
+    Get,
+    Res,
+    Logger,
+    Req, Post,
+} from '@nestjs/common';
+import { ApiUseTags, ApiResponse } from '@nestjs/swagger';
+import { FacebookFeedService } from '../facebook-feed/facebook-feed.service';
+import { ConfigService } from '../config/config.service';
+import { TwitterFeedService } from '../twitter-feed/twitter-feed.service';
+const config = new ConfigService();
+const crypto = require('crypto')
+
+@ApiUseTags('webhook')
+@Controller('/api/webhook')
+export class WebhookController {
+    constructor(
+        private readonly facebookFeedService: FacebookFeedService,
+        private readonly twitterFeedService: TwitterFeedService,
+        private readonly logger: Logger) {
+        this.logger = new Logger('WebhookController');
+    }
+
+    @Get('facebook')
+    @ApiResponse({status: 200, description: 'Verification endpoint for facebook fetch'})
+    async fbVerify(@Req() request, @Res() response) {
+        try {
+            this.logger.log('Facebook Verification challenge', request.query['hub.challenge']);
```

Comment:
Did you verify that this is working properly?. Do you need to change this to something like? this.logger.log(`Facebook Verification challenge' : ${request.query['hub.challenge']}`);